### PR TITLE
Added firmware driver resource

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added firmware driver resource

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -78,6 +78,12 @@
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PUT      | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | DELETE   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/bulk-delete</sub>                                               | POST     | :white_check_mark:   | :white_check_mark:   |
+|     **Firmware Drivers**                                                                                                                         |
+|<sub>/rest/firmware-drivers</sub>                                                        | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```
Comment (AsisBagga) [inline endpoints-support.md:82]: havent we validated this against 2400 and 2200?

**Review by AsisBagga — APPROVED** [review]

File: tests/unit/test_oneview_firmware_driver_facts.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import pytest
+from hpe_test_utils import OneViewBaseFactsTest
```
Comment (SHANDCRUZ) [inline tests/unit/test_oneview_firmware_driver_facts.py:22]: ansible_collections.hpe.oneview.tests.unit.utlis. is not included.

**Review by SHANDCRUZ — APPROVED** [review]

File: endpoints-support.md (hunk 1 above)
Comment (VenkateshRavula) [reply endpoints-support.md]: changed

File: tests/unit/test_oneview_firmware_driver_facts.py (hunk 2 above)
Comment (VenkateshRavula) [reply tests/unit/test_oneview_firmware_driver_facts.py]: done

**Pull request merged**